node/consensus/src/lib.rs

```diff
@@ -359,7 +359,7 @@ impl<N: Network, C: ConsensusStorage<N>> Consensus<N, C> {
             self.memory_pool.clear_invalid_solutions(self);
         }
 
-        info!("Advanced to block {}", block.height());
+        debug!("Advanced to block {}", block.height());
```

Comment:
don't we want to maintain this level? we should be able to see that a block was advanced in every node

Reply:
Well, I'm not sure, you can still see it if you want (set verbosity to 1). Not a big deal to me though, this is just one log line per 30 seconds or so.

Reply:
yeah, I think I'd keep this as `info` as it's not spammy and it's an interesting piece of information; all the others LGTM